.buildconfig/macos-latest.yml

```diff
@@ -4,25 +4,7 @@ channels:
 dependencies:
   - cmake
   - conan>=1.45.0
-  - confuse
-  - graphlib-backport
-  - h5py
-  - hypothesis
-  - ipympl<0.9
-  - ipywidgets
-  - markupsafe>=1.1.1,<2.1.0
```

Comment:
Was the `markupsafe` issue definitely fixed with `conan>=1.45`?

Reply:
It works now. Was something silently broken?